data/campaigns/Under_the_Burning_Suns/units/quenoth/Tauroch.cfg

```diff
@@ -15,7 +15,7 @@
     advances_to=null
     cost=12
     usage=fighter
-    description= _ "Taurochs are huge, hardened beasts of the desert, considered untamable by most surviving races. However, the elves’ affinity with nature has allowed them to form an unlikely bond with these unruly animals. While Taurochs can easily carry or pull heavy loads, their thick hides and unwavering resolution also make them formidable mounts in battle."
+    description= _ "Taurochs are huge, hardened beasts of the desert, considered untamable by most surviving races. However, the elves’ affinity with nature has allowed them to form an unlikely bond with these unruly animals. While taurochs can easily carry or pull heavy loads, their thick hides and unwavering resolution also make them formidable mounts in battle."
```

Comment:
Commit c4492327306cfdd81c34848219e57cb738868788 also seems to be here in error.

Reply:
Also this should be fixed as well.